server/app-engine/stress-test.py

```diff
@@ -133,41 +136,34 @@ def main(argv):
 	intervalVariation = 1
 	# How many updates per agency
 	numRepeats = 1
-	domain = None
-
-	for i in range(argc):
-	    arg = sys.argv[i]
-
-	    if arg == '-l':
-	        domain = 'https://127.0.0.1:8080/'
-
-	    if arg == '-p':
-	        domain = 'https://lat-long-prototype.wl.r.appspot.com/'
-
-	    if arg == '-ac' and i < argc - 1:
-	        i += 1
-	        agencyCount = int(sys.argv[i])
+	domain = LOCAL_SERVER_URL
 
-	    if arg == '-vc' and i < argc - 1:
-	        i += 1
-	        vehicleCount = int(sys.argv[i])
-
-	    if arg == '-i' and i < argc - 1:
-	        i += 1
-	        intervalTime = int(sys.argv[i])
-
-	    if arg == '-iv' and i < argc - 1:
-	        i += 1
-	        intervalVariation = int(sys.argv[i])
+	if argc != 2:
+		print('* usage: python stress-test.py <path-to-config-json-file>')
+		exit(1)
 
-	    if arg == '-r' and i < argc - 1:
-	        i += 1
-	        numRepeats = int(sys.argv[i])
+	jsonInput = sys.argv[1]
 
-	if domain == None:
-		print('* usage: use -l for local server or -p for production server')
+	if '.json' not in jsonInput:
+		print('* error: input must be a JSON file')
 		exit(1)
 
+	f = open(jsonInput)
+	data = json.load(f)
+	f.close()
+
+	if data.get('production-server', False):
+		domain = PRODUCTION_SERVER_URL
+	else:
+		domain = LOCAL_SERVER_URL
```

Comment:
Just a thought, but wouldn't it be more flexible if you just had a `server_url` attribute in the json? If not set, it defaults to `localhost`. That way, if we ever have a real staging env we can easily point to that, and also other people using the repo can point to whatever servers they have set up.

Reply:
That's a nice idea and will add, thanks